advanced_functionality/pytorch_deploy_large_GPT_model/GPT-J-6B-model-parallel-inference-DJL.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #28.    docker tag &lt;your image id&gt;&nbsp; ${fullname}
</pre><p>Is it possible to programmatically retrieve the image ID so users don't have to manually copy and paste it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3540/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
@billdoors completed this